.github/renovate.json5

```diff
@@ -6,12 +6,29 @@
     enabled: true,
   },
   separateMajorMinor: false,
+  ignoreDeps: ["@types/sortablejs"], // we use a static version
   packageRules: [
```

Comment:
Django updates are more involved than just clicking "accept" on a PR. Often, we have to wait for support of the new version from other dependencies. During that time, having a PR open doesn't really help I think. If it's grouped together with the other python dependencies, that makes it harder to single it out as "cannot yet upgrade". I think it would make sense to put it in a separate group, and maybe also have it only trigger manually?